File: hackerrank/easy/0045-birthday-cake-candles/python/solution.py
## solution.py

from typing import List

def birthdayCakeCandles(candles: List[int]) -> int:
    if not candles:
        return 0
    tallest = max(candles)
    return candles.count(tallest)

# --- demo run ---
if __name__ == "__main__":
    print(birthdayCakeCandles([3, 2, 1, 3]))   # Expected: 2
    print(birthdayCakeCandles([4, 4, 1, 3, 4]))  # Expected: 3

File: hackerrank/easy/0045-birthday-cake-candles/swift/solution.swift
// solution.swift

import Foundation

func birthdayCakeCandles(candles: [Int]) -> Int {
    guard let tallest = candles.max() else { return 0 }
    return candles.filter { $0 == tallest }.count
}

// --- demo run ---
print(birthdayCakeCandles(candles: [3, 2, 1, 3]))  // Expected: 2
print(birthdayCakeCandles(candles: [4, 4, 1, 3, 4]))  // Expected: 3